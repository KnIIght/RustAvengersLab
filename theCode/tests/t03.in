0
Bruce
being angry
my fists
