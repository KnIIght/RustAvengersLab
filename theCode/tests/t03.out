My name is Bruce, my power is being angry and my weapon is my fists.
HULK SMASH!
In all due time. A hero becomes more than a hero.
